sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -232,9 +283,9 @@ export class CertificateClient {
 
   /**
    * Creates an instance of CertificateClient.
-   * @param {string} vaultUrl the base URL to the vault.
-   * @param {TokenCredential} credential An object that implements the `TokenCredential` interface used to authenticate requests to the service. Use the @azure/identity package to create a credential that suits your needs.
-   * @param {PipelineOptions} [pipelineOptions={}] Optional. Pipeline options used to configure Key Vault API requests.
+   * @param vaultUrl the base URL to the vault.
+   * @param credential An object that implements the `TokenCredential` interface used to authenticate requests to the service. Use the @azure/identity package to create a credential that suits your needs.
+   * @param [pipelineOptions={}] Optional. Pipeline options used to configure Key Vault API requests.
```

Comment:
Revert the changes to jsdocs in this section.
The `@param [pipelineOptions={}]` part doesnt look right.

Its preferable if this PR only focuses on name change and the occasional text changes in jsdocs and not the format.
We can go over the format as a team across libraries separately later

Reply:
is `@param {type}` valid format? I tried looking at the tsdocs and I couldn't find it. `pipelineOptions={}` was not something I added as part of this PR.

Reply:
`pipelineOptions={}` was not something you added as part of this PR.
But `@param {PipelineOptions} [pipelineOptions={}]` did get changed to `[pipelineOptions={}]` in this PR.

 `@param {PipelineOptions} [pipelineOptions={}]` is valid, I dont know if `pipelineOptions={}` is valid.

What I am saying is that the removal of {type} can be kept out of the scope of this PR to reduce uncertianities

Reply:
I'll undo it! I just couldn't find `@param {type} [optionalvalue=defaultvalue]` in the tsdocs.

Reply:
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
fixed!